debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/AbstractLogMinerEventProcessor.java

```diff
@@ -237,10 +238,12 @@ public Scn process(Scn startScn, Scn endScn) throws SQLException, InterruptedExc
                         metrics.getSleepTimeInMilliseconds());
 
                 if (metrics.getNumberOfActiveTransactions() > 0) {
```

Comment:
Could change this to 

`if (metrics.getNumberOfActiveTransactions() > 0 && LOGGER.isDebugEnabled()) {`

to avoid iterating over the cache when not needed.

Reply:
Should also review the other cache usages, pointed out [here](https://debezium.zulipchat.com/#narrow/stream/348250-community-oracle/topic/Poor.20performance.20when.20using.20infinispan.20caches/near/397111756).